frontend/public/components/import-yaml-results.tsx

```diff
@@ -0,0 +1,299 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { Helmet } from 'react-helmet';
+import { Location } from 'history';
+import { useTranslation } from 'react-i18next';
+import * as _ from 'lodash';
+import { Map as ImmutableMap } from 'immutable';
+import { Bullseye, Button, Spinner } from '@patternfly/react-core';
+import {
+  GreenCheckCircleIcon,
+  RedExclamationCircleIcon,
+  YellowExclamationTriangleIcon,
+} from '@console/shared';
+import {
+  TableComposable,
+  TableGridBreakpoint,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+} from '@patternfly/react-table';
+
+import { history, ResourceLink } from './utils';
+import { createRequests } from './edit-yaml';
+import { K8sKind, K8sResourceKindReference } from '../module/k8s';
+import { formatNamespacedRouteForResource } from '@console/shared/src/utils';
+
+const reactPropFix = {
+  translate: 'no',
```

Comment:
Can you describe why this is needed?

Reply:
@spadgett added explanation comment taken from pipelines code.